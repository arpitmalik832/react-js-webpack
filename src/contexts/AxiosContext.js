import { createContext, useContext } from 'react';

const AxiosContext = createContext();

const useAxiosInstances = () => useContext(AxiosContext);

export { AxiosContext, useAxiosInstances };
