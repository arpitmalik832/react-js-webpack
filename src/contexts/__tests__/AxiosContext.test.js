import { cleanup, renderHook } from '@testing-library/react';
import '@testing-library/jest-dom';

import { useAxiosInstances } from '../AxiosContext';

describe('AxiosContext unit tests', () => {
  afterEach(() => {
    cleanup();
  });

  it('AxiosContext snapshot test', () => {
    const { result } = renderHook(useAxiosInstances);

    expect(result).toMatchSnapshot();
  });
});
