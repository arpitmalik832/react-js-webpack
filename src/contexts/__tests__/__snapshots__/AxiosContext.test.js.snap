// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AxiosContext unit tests AxiosContext snapshot test 1`] = `
{
  "current": undefined,
}
`;
